Lib/test/test_asyncio/test_tasks.py

```diff
@@ -245,32 +241,6 @@ def notmuch():
         self.assertTrue(t.done())
         self.assertEqual(t.result(), 'ok')
 
-        a = notmuch()
```

Comment:
Why so many code was removed? Should not only some `assertWarns()` be removed?

Reply:
Good catch, I must have merged two conflict resolutions into one by accident. I'll fix that up right now with the `s/3.11.1/3.10.9/` changes.